pages/ajax/searches/searchmessages.php

```diff
@@ -3,4 +3,18 @@
 // Connect to the db
 file_exists('../../../../mysqli_connect.inc.php') ? require_once('../../../../mysqli_connect.inc.php') : require_once('../../../../Users/VSpoe/mysqli_connect.inc.php');
 
-?>
\ No newline at end of file
+$searchq = "SELECT * FROM messages WHERE MATCH (subject, body) AGAINST (\"{$_POST['q']}\")";
+
+$searchr = mysqli_query($dbc, $searchq);
+$searchrows = mysqli_fetch_all($searchr, MYSQLI_ASSOC);
+
+$searchresult = "";
```

Comment:
This is unnecessary: just echo it.

Reply:
I do not think this works, I think I have tried it before, and it only echoed the last piece of text

Reply:
The scenario might have been different, but I know for sure, as I have tested it, that it will work if you completely removed $searchresult, and replace 
`$searchresult .= ....` 
with
`echo ....`

Reply:
Thank you for the feedback @GnatSwatter. Though I believe you, I will keep it the way it is for consistency, as I did not know this in the working of the previous ajax files.